### PR TITLE
Multiple items and image support

### PR DESCRIPTION
**Pull request by samwarnick**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -19,37 +24,49 @@ class RSS extends HTMLElement {
...
+		const description = item.querySelector('description')?.textContent || item.querySelector('summary')?.textContent
+		const pubDate = item.querySelector('pubDate')?.textContent || item.querySelector('updated')?.textContent
+		const hostname = link ? new URL(link).hostname : 'No hostname'
+ 		const userLocale = navigator.language || 'en-US'
+		const formattedPubDate = new Date(pubDate).toLocaleDateString(userLocale, {
+			year: 'numeric',
+			month: 'long',
+			day: 'numeric'
+		})
+		const mediaContent = item.querySelector('content')
```
Comment (samwarnick) [inline index.js:55]: Probably possible there are multiple items

File: index.js [hunk 2]
```diff
@@ -19,37 +24,49 @@ class RSS extends HTMLElement {
...
+		const mediaIsImage = mediaContent?.getAttribute('medium') === 'image'
```
Comment (samwarnick) [inline index.js:56]: Could add support for video and audio as well

File: index.js [hunk 3]
```diff
@@ -1,12 +1,17 @@
 class RSS extends HTMLElement {
 	async connectedCallback() {
 		const feedUrl = this.getAttribute('url')
+		let postCount = parseInt(this.getAttribute('count'))
```
Comment (samwarnick) [inline index.js:4]: Don't know if `count` is a good name

**Comment by catskull** [comment]
plz add tests

**Comment by catskull** [comment]
jk jk

**Comment by samwarnick** [comment]
> plz add tests

What are tests? Plz advise.

**Pull request merged**